src/components/Heading/src/Heading.vue

```diff
@@ -119,6 +119,7 @@ export default {
 
 .Heading {
 	margin: 0;
+	color: var(--color-900);
```

Comment:
Hmmm, I'm wondering what this might look like in the near future one we add color props to MHeading and allow local overrides... maybe it'll be `color: var(--text-color, var(--color-900, #000))` where `--text-color` is some CSS var computed locally... also, do we need a default here, e.g. `var(--color-900, #000)`?

Reply:
This might be confusing due to the fact that this is a generated value, but we need to consider that our existing profiles allow explicit customization of the heading color that's not tied to the background color.